### ISSUE TITLE
Functional difference between Chart values.yaml and values-crd.yaml

### ISSUE THREAD

**Issue by apeschel** [zalando/postgres-operator#1343]
Please, answer some short questions which should help us to understand your problem / question better?

- **Which image of the operator are you using?** head of master
- **Where do you run it - cloud or metal? Kubernetes or OpenShift?** N/A
- **Are you running Postgres Operator in production?** no
- **Type of issue?** bug report

The following keys are set in the values-crd.yaml, and it's not clear why, since they don't seem to be relevant to the CRD based installation.

```
pam_role_name: zalandos
protected_role_names:
- admin
team_admin_role: admin
team_api_role_configuration:
  log_statement: all
```

**Comment by FxKu** [comment]
Why shouldn't they be relevant? :smiley:
Okay, the options are commented out in the values files and manifest examples - but if you want to use them - you can. Consider the commented parts examples so that users understand how to configure the operator.

**Comment by apeschel** [comment]
@FxKu They're commented in `values.yaml`, but they're not commented out in the `values-crd.yaml`, which means the CRD based install adds some additonal role stuff that the base install doesn't.

This isn't necessarily a problem, but it's definitely inconsistent.

**Comment by FxKu** [comment]
We've now merged (#1224) the two values.yaml files.
